circle.yml

```diff
@@ -1,18 +1,29 @@
 machine:
   pre:
     - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
+    - pip install docker-compose
   services:
     - docker
 
+dependencies:
+  override:
+    - docker pull selenium/standalone-firefox
+    - docker pull tomtoothfairies/chiron:0.3
+
 compile:
   override:
-    - docker build --rm=false -t panacea panacea
-    - docker build --rm=false -t asclepius asclepius
+    - docker build --rm=false -t tomtoothfairies/panacea panacea
+    - docker build --rm=false -t tomtoothfairies/asclepius asclepius
+    - docker build --rm=false -t tomtoothfairies/athloi athloi
 
 test:
   pre:
     - mkdir -p $CIRCLE_TEST_REPORTS/exunit
     - mkdir -p $CIRCLE_TEST_REPORTS/pytest
+    - mkdir -p $CIRCLE_TEST_REPORTS/cucumber
+
   override:
-    - docker run -t -e "MIX_ENV=test" -e "CI=true" -v $CIRCLE_TEST_REPORTS/exunit:/test-reports panacea mix test
-    - docker run -t -v $CIRCLE_TEST_REPORTS/pytest:/test-reports asclepius pytest --junitxml=/test-reports/test-junit-report.xml
+    - docker-compose -f docker-compose.e2e.yml -f docker-compose.e2e.ci.yml up -d
```

Comment:
Yeah so I want to give him the option to be able to run these tests locally, but also in CI I want overrides to have a volume to collect junit xml.